unfoldingWord/translationCore
# Fixes For Stepper Colors

#### This pull request addresses:
This PR makes the colors at the top when navigating through the stepper more consistent.


#### How to test this pull request:
Ensure the the following is always true no matter what step you are on.
The current step is black
The previous steps are blue
The remaining steps are grey


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3156)
